src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/Properties/launchSettings.json

```diff
@@ -16,7 +16,7 @@
         "ASPNETCORE_HTTPS_PORT": "44300"
       }
     },
-    "Company.WebApplication1": {
+    "AureliaSpa": {
```

Comment:
To help deal with when the user inputs a name for a template that is not valid in some contexts (namespace for example) template processing attempts to create valid variations on the input name, so that the generated template content will build. But to do this, the sourceName must be in a format that will be properly dealt with when applying the variations. It's for this reason that we generally use a name like "Company.WebApplication1".

So I'd recommend making the change in the opposite direction, that is, leave the source name as Company.WebApplication1 in the template.json, and change "AureliaSpa" to "Company.WebApplication1" in the template content files. Otherwise when users enter names that aren't valid symbols in the context where they get replaced, the generated template won't build.

(Same thing for all the other templates with similar changes in this PR).